README.md

```diff
@@ -1,7 +1,7 @@
 This module allows you to easily extend the built-in Error type for typed error checking
 
 ```coffee-script
-TypedError = require 'typed-error'
+{ TypedError } = require 'typed-error'
 
 class MyError extends TypedError
```

Comment:
I added an additional example for typescript (which is very close to es6), which I think makes more sense since we now offer all of the typescript typing (and also I know it works and is in the tests, which is a plus ;) )